app/views/shared/_terms_of_deposit.html.erb

```diff
@@ -5,20 +5,24 @@
           <div class="col-5">
             <h1 class="modal-title h3" id="terms-of-deposit-prompt">Terms of Deposit</h1>
           </div>
-          <div class="col-2">
-            <%= link_to Settings.faq_url, aria: {label: "FAQs about Terms of Deposit"}, id: "faq-link", class: "text-decoration-none", target: "_blank" do %>
-              <span class="fa-solid fa-question-circle"></span> FAQs
-            <% end %>
-          </div>
-          <div class="col-2">
-            <%= link_to Settings.terms_url, aria: {label: "Save Terms of Deposit"}, id: "print-link", class: "text-decoration-none", target: "_blank" do %>
-              <span class="fa-solid fa-save"></span> Save
-            <% end %>
-          </div>
-          <div class="col-2">
-            <%= link_to print_terms_of_deposit_path, aria: {label: "Print Terms of Deposit"}, id: "save-link", class: "text-decoration-none", target: "_blank" do %>
-              <span class="fa-solid fa-print"></span> Print
-            <% end %>
+          <div class="col-6">
+            <ul class="row list-unstyled">
+              <li class="col-4">
+                <%= link_to Settings.faq_url, aria: {label: "FAQs about Terms of Deposit"}, id: "faq-link", class: "text-decoration-none", target: "_blank" do %>
+                  <span class="fa-solid fa-question-circle"></span> FAQs
+                <% end %>
+              </li>
+              <li class="col-4">
+                <%= link_to Settings.terms_url, aria: {label: "Save Terms of Deposit"}, id: "print-link", class: "text-decoration-none", target: "_blank" do %>
+                  <span class="fa-solid fa-save"></span> Save
+                <% end %>
+              </li>
+              <li class="col-4">
+                <%= link_to print_terms_of_deposit_path, aria: {label: "Print Terms of Deposit"}, id: "save-link", class: "text-decoration-none", target: "_blank" do %>
+                  <span class="fa-solid fa-print"></span> Print
+                <% end %>
+              </li>
+            </ul>
           </div>
```

Comment:
Couple quick questions:

1. I naïvely assumed each `li` would wind up with a `col-2` class given the containing `div` is a `col-6`
2. Did you find that the new containing `div` is needed to retain the current look & feel?

Reply:
@aaron-collier Ah, OK, makes sense. Thanks. 🙂